.travis.yml

```diff
@@ -5,11 +5,9 @@ branches:
     - /\bv[0-9]/
 dist: xenial
 python:
-  - "3.4"
-  - "3.5"
-  - "3.6"
   - "3.7"
```

Comment:
why don't we want to support other versions of python3? Just 3.7? We are limiting too much to use 3.7, where there are some OS with only 3.6 available for example..

Reply:
ok maybe with excluding 3.6 it is too much , but for sure we should drop 3.4 because it is out of support. With 3.5 I also not sure why we need waste our time on it. Import point here that each version increasing waiting time  for Travis to finish. Sometimes it make huge difference ...

Reply:
Ok, let's keep 3.6 and 3.7 then. 3.6 is very well extended, but 3.7 not :)